spec/lib/request_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+
+describe Request do
+  before do
+    env = Rack::MockRequest.env_for(
+      '/coveralls',
+      'REQUEST_METHOD' => 'POST',
+      'HTTP_USER_AGENT' => 'curl/7.43.0',
+      'HTTP_ACCEPT' => '*/*',
+      'CONTENT_TYPE' => 'application/x-www-form-urlencoded',
+      :input => 'foo=bar&quux=bla'
+    )
+
+    @req = Request.new(env: env)
+  end
+
+  it '#method equal POST' do
+    assert { @req.method == 'POST' }
+  end
+
+  it '#path equal /coveralls' do
+    assert { @req.path == '/coveralls' }
+  end
+
+  it '#headers equal /coveralls' do
+    headers = {
+      'User-Agent' => 'curl/7.43.0',
+      'Content-Type' => 'application/x-www-form-urlencoded',
+      'Accept' => '*/*'
+    }
+
+    assert { @req.headers == headers }
+  end
+
+  it "#body" do
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols.